### DISCUSSION TITLE
Form fields have default browser opacity, should this be overridden?

### DISCUSSION THREAD

**Discussion by garand** [tailwindlabs/tailwindcss#2887] [Ideas]
I recently discovered when building out my component library that form elements have default CSS setting the opacity or opacity of the background (select vs input) when the fields are marked as disabled. I ended up overriding the default disabled styling for form fields in my global CSS so I can have better control over it with Tailwind. Does it make sense to set a global default for disabled form fields in `tailwindcss/base` so that these fields can be a little more easily managed?

https://codepen.io/garand/pen/VwKZKNy

**Comment by MathGeniusJodie** [comment]
it's worth keeping in mind that disabled fields are a bit of an anti-pattern https://medium.com/@izzmo/anti-pattern-using-disabled-or-read-only-for-form-input-e2c24c669f5b I don't think their use should be encouraged by the library
